### PR TITLE
TEAM TEACHING - first cut at allowing entities to add/remove their own Scene Items

### PR DESCRIPTION
**Pull request by ZappoMan**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 2fba5a0928e943f8192c2abd25949abb6ae4cb1c available at: https://gist.github.com/00cee886533ada4b260c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4989.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4989.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 1029a8af7c7cf7609f7a4b1c49e943552dc19632 available at: https://gist.github.com/00cee886533ada4b260c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4989.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4989.dmg

File: libraries/entities-renderer/src/RenderableLightEntityItem.h [hunk 1]
```diff
@@ -22,6 +22,7 @@ class RenderableLightEntityItem : public LightEntityItem  {
         LightEntityItem(entityItemID, properties)
         { }
 
+    virtual bool canRenderInScene() { return false; } // we don't yet play well with others
```
Comment (samcake) [inline libraries/entities-renderer/src/RenderableLightEntityItem.h]: Actually you could not worry about this one, we are talking care of that

**Comment by samcake** [comment]
OK, i can merge

**Pull request merged**